src/ui/components/confirm_gesture.c

```diff
@@ -72,7 +72,7 @@ static void _render(component_t* component)
 
     // Draw the top arrow
     const uint16_t padding = 1;
-    x = SCREEN_WIDTH * 15 / 16 - (arrow_height * 2 + 1);
+    x = SCREEN_WIDTH * 15 / 16 - (arrow_height * 1.5);
```

Comment:
how about `arrow_height + arrow_height / 2`?

Reply:
now i see why niklas did this (this is how he did in in his PR). will do